commands/create-project-api.js

```diff
@@ -88,8 +71,7 @@ async function getFileFromCore(fileName, rootDir = reactionAppRoot) {
 function updateEnv(envData) {
   const env = parse(envData);
   env.MONGO_URL = "mongodb://localhost:27017/reaction";
-  const updatedEnv = stringify(env);
-  return updatedEnv;
+  return stringify(env);
```

Comment:
I really dislike this style (process and return in one) since it makes it debugging much more difficult